src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirCosmosDocumentQueryFactory.cs

```diff
@@ -6,24 +6,25 @@
 using EnsureThat;
 using Microsoft.Azure.Documents;
 using Microsoft.Azure.Documents.Linq;
+using Microsoft.Health.CosmosDb.Features.Storage;
 using Microsoft.Health.Fhir.Core.Features.Context;
 
 namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage
 {
     /// <summary>
     /// Factory for creating the <see cref="FhirDocumentQuery{T}"/>.
     /// </summary>
-    public class CosmosDocumentQueryFactory : ICosmosDocumentQueryFactory
+    public class FhirCosmosDocumentQueryFactory : ICosmosDocumentQueryFactory
```

Comment:
There seems to be inconsistency here. This one was changed to FhirCosmosDocumentQueryFacctory but should CosmosHealthCheck, CosmosSearchService, and etc also be renamed to FhirCosmosHealthCheck and so forth?

Reply:
Yes, I think it would make sense to work out the inconsistencies here.

Reply:
All of the relevant files in the Fhir.CosmosDb project should be renamed now.